### PR TITLE
Aura 13

### PR DESCRIPTION
**Pull request by ZipengChen1832**
Added resuable card component

### CODE REVIEW

File: src/components/card/_card.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.card {
+    padding: 1rem;
+    display: grid;
+    place-items: center;
+    border-radius: 1rem;
```
Comment (GoldenaArcher) [inline src/components/card/_card.scss]: Please clean up the code here

File: src/ui/card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React, {ReactElement} from 'react'
+import { Card as MUICard, Paper } from '@mui/material';
+import './_card.scss'
+
+enum TYPE  {
+
```
Comment (GoldenaArcher) [inline src/ui/card/index.tsx]: During out conversation I mentioned that implement transparent background as shown in this image

transparent

**Review by GoldenaArcher — CHANGES_REQUESTED** [review]
First delete redundant code, and add transparent feature, you can work on other feature later.

Also I can see you haven't pull the latest code from main branch, I have done some configuration to sync the coding style. please do so as well.

AND, Please don't leave that many warning in the console.

File: src/components/card/_card.scss (hunk 1 above)
Comment (ZipengChen1832) [reply src/components/card/_card.scss]: This section centers the content inside card, as well as styling the border's radius, I don't know what else is expected for the component

Comment (ZipengChen1832) [reply src/components/card/_card.scss]: As for the transparent feature, I'm expecting the user to use rgba() when passing the backgroundColor prop

Comment (GoldenaArcher) [reply src/components/card/_card.scss]: image

1. there are 2 copies of cards.

2. for the props, did you check the usage of `enum` ?

3. Your code is not synced with the latest code from `main` branch, I have implemented the code formatter, but it's not used.

File: src/ui/card/_card.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
+
+$gray-100: #939393 !default;
+$blue-400-transparent: #186c9666 !default;
+
+.card {
+    padding: 1rem;
+    display: grid;
+    place-items: center;
+    border-radius: 1rem;
```
Comment (GoldenaArcher) [inline src/ui/card/_card.scss]: image

image


border radius is off

File: src/ui/card/index.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  className?: string;
+  children: ReactElement;
+  backgroundColor?: string;
+  elevation?: number;
+  shadow?: boolean;
+  transparent?: boolean;
+};
+
+const Card = ({
+  className = '',
```
Comment (GoldenaArcher) [inline src/ui/card/index.tsx:15]: image

transparent not working

**Review by GoldenaArcher — CHANGES_REQUESTED** [review]

**Pull request merged**